MiniBootcamp/MiniBootcampTests/FeedDataManagerTests.swift

```diff
@@ -0,0 +1,121 @@
+//
+//  FeedDataManagerTests.swift
+//  MiniBootcampTests
+//
+//  Created by Chioma Amanda Mmegwa  on 29/05/2023.
+//
+
+import XCTest
+@testable import MiniBootcamp
+
+class FeedDataManagerTests: XCTestCase {
```

Comment:
Maybe we can declare this class as final. Also leave and empty line after its declaration